### ISSUE TITLE
change input channel on device?

### ISSUE THREAD

**Issue by chris-ampd** [Maxx53/NoCableLauncher#29]
Hi, 
this is working perfectly with my focusrite saffire 56 using channel 1 as my input channel. However.. I would like to use a different channel, is this possible and how would I go about doing this?

**Comment by Maxx53** [comment]
Tecnically that possible, but you'll get huge delay in practice.
https://github.com/Maxx53/NoCableLauncher/issues/11

**Comment by chris-ampd** [comment]
ok, I just had a read of #11. So I gather it's not possible to find a new channel on the device by altering the VID/PID values?

**Comment by Maxx53** [comment]
No, multiple-input audio device (as any device) has one PID/VID.
